src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function name `calculateRentalCost` is descriptive and adheres to checklist item 1.